system/Config/Services.php

```diff
@@ -585,7 +589,21 @@ public static function session(?App $config = null, bool $getShared = true)
         $logger = AppServices::logger();
 
         $driverName = $config->sessionDriver;
-        $driver     = new $driverName($config, AppServices::request()->getIPAddress());
+
+        if ($driverName === DatabaseHandler::class) {
```

Comment:
how about using `is_a()` check?

```suggestion
        if (is_a($driverName, DatabaseHandler::class, true)) {
```

for handle extended DatabaseHandler ?

Reply:
Thanks for suggestion, but in this case, I think string comparison is enough.
Because it really compares just strings. It has nothing to do with the parent class.

Reply:
ok 👍